middle.mkd

```diff
@@ -274,8 +274,9 @@ A GeoJSON object with the type "Feature" is a feature object.
 A GeoJSON object with the type "FeatureCollection" is a feature
 collection object. A feature collection object MUST have a member with
 the name "features". The value of "features" is an array. Each element
-of the array is a feature object as defined above. It is possible for
-this array to be empty.
+of the array MUST be a feature object as defined above. It is possible
+for this array to be empty. The values in the array are ordered, meaning
+that GeoJSON implementations MUST maintain order when processing them.
```

Comment:
@dret without challenging your motivations, I want to argue that we don't need to add that constraint.

I can imagine different ordering being useful to different kinds of applications, and I'd like to be able to freely sort as I need within my data processing pipelines. In other words, I would flaunt that MUST in some parse-reserialize situations, and I think other developers would also flaunt it.

I'm also not aware of any JSON parsers that give access to array elements in other than their JSON order, so this MUST looks well enforced already.